django_tutorial/first_project/sba_app/models.py

```diff
@@ -0,0 +1,3 @@
+from django.db import models
+
+# Create your models here.
```

Comment:
I have first_project/firstapp powering the API, so it has all the Django models.  first_project/sba_app is the frontend app; on the Django side it's just some static HTML and JS, and then the JS consumes the API on the frontend.